sass/components/_scrollbar.scss

```
@@ -19,7 +19,7 @@ body {
         scrollbar-3dlight-color: #7D7E94;
         scrollbar-arrow-color: #C1C1D1;
         scrollbar-darkshadow-color: #2D2C4D;
-        scrollbar-face-color: rgba(0, 0, 0, .1);
+        scrollbar-face-color: rgba(var(--center-channel-bg-rgb), .1);
```

@tony-ho, looks like the theme override doesn't use transparency, so this might need to be changed to:

```suggestion
        scrollbar-face-color: var(--center-channel-bg);
```

---

Thanks @deanwhillier, I've made the suggested change.